test/Tests/Readers/Markdown.hs

```diff
@@ -307,6 +307,14 @@ tests = [ testGroup "inline code"
             "[https://example.org(](url)" =?>
             para (link "url" "" (text "https://example.org("))
           ]
+        , testGroup "Github wiki links"
+          [ test markdownGH "autolink" $
+            "[[https://example.org]]" =?>
+            para (link "https://example.org" "" (text "https://example.org"))
+          , test markdownGH "link with title" $
+            "[[title|https://example.org]]" =?>
+            para (link "title" "" (text "https://example.org"))
```

Comment:
maybe add a test for e.g.

    [[Link Text|nameofwikipage]]

and one for:

    [[Name of Page]]

(assuming that's the syntax for pages with spaces in them)

Reply:
Another good test (as noted above) wolud be `[[Name of page] and]]`